administration/index.md

```diff
@@ -13,5 +13,6 @@ google-workspace
 github
 airtable
 authentication
+passwords
```

Comment:
This is fixing a bug where we forgot to add the passwords page in the toctree for this section